### ISSUE TITLE
[Bug]: [1.19.2 Forge] jei-config.ini doesn't change column or row count.

### ISSUE THREAD

**Issue by WilliamTDoran** [mezz/JustEnoughItems#3118]
### Steps to Reproduce the Bug

1. Change column or row count in jei-config.ini
2. Open jei ingame
3. Observe that the column and row count remain at the default 9 and 16.

### Expected Behavior

Ingame column and row should match what's been set in jei-config.ini

### Actual Behavior

Nothing changes

**Comment by Deceron** [comment]
same issue here with me and 3 of my friends.

**Comment by NeosGhost** [comment]
same here version 11.5.2.1000
does not seem like it is pulling any of the client-side config options.

**Comment by mezz** [comment]
Thanks for the report!
What is the full location of the config file you are editing? 
Can you try deleting the `jei/` folder in your configs and letting JEI recreate the files?

**Comment by WilliamTDoran** [comment]
The folder is config/jei/jei-client.ini

Recreating the config produced no change in behavior

**Comment by mezz** [comment]
Is this happening with Forge or Fabric?
I'll try it out to see if I can find the issue.

**Comment by mezz** [comment]
on Forge, the client configs seem to work fine for me 
![2023-01-29_18 11 09](https://user-images.githubusercontent.com/916092/215373187-43a3f5f0-0794-4b5d-9a35-74c1c8177327.png)

**Comment by mezz** [comment]
Fabric is also working
![2023-01-29_18 13 54](https://user-images.githubusercontent.com/916092/215373385-c2c7a102-d8a6-48ae-9bcc-915db34a6ac8.png)

**Comment by mezz** [comment]
I need some more information to solve this issue, please paste your full `fml-client-latest.log` or `latest.log` to https://gist.github.com and link it here. Maybe it'll show some error with loading the config file changes.

**Comment by WilliamTDoran** [comment]
This is on forge 43.2.1

https://gist.github.com/WilliamTDoran/a7f5b125e1406f43d9ac73f514a3c4a0

**Comment by mezz** [comment]
Thanks for the log.
I can see it is loading the config file, can you confirm that you are editing the file at this exact location?
`Loading config file: C:\Users\eagle\curseforge\minecraft\Instances\Druid's Life\config\jei\jei-client.ini`

**Comment by WilliamTDoran** [comment]
Can confirm

**Comment by NeosGhost** [comment]
did some fiddling with the config. so the search bar location changed like expected but the item list did not get any additional columns.
![image](https://user-images.githubusercontent.com/16095247/215613920-f45288c8-488f-4296-8732-ec508f51a672.png)

if i shrink my screen the list does collapse to fewer columns but i cant seem to get it to display any more than 9 even with it stretched across both my 2k screens.

**Comment by mezz** [comment]
There are configs for the ingredient list and the bookmarks list in that file, are you editing the one for the bookmarks list?

**Comment by WilliamTDoran** [comment]
> 

...yes I was

Thank you lmao

**Comment by mezz** [comment]
Haha nice alright, thanks for confirming.
Maybe I should split this into multiple config files.

**Comment by WilliamTDoran** [comment]
Might be helpful to include what field's being edited in the # Description. I know it says like literally one line above, but when google has said you're looking for the value called MaxColumns, you tend to just beeline for MaxColumns

**Comment by mezz** [comment]
Yeah I think it's bad design philosophy to blame the players when something like this goes wrong. 
I'd like it to be easy to use even if you're not paying close attention.
It may be about time for me to make this more configurable in-game.

**Comment by NeosGhost** [comment]
same for me. i promise i can read xD

**Comment by Etarnax** [comment]
I'm getting the same issue as reported before with some slight tweaks.
Mod Version: jei-1.19.2-forge-11.6.0.1013
Forge Version: forge-1.19.2-43.2.11

When I first launched I had old config files still in the config folder and the numbers for the rows and columns were not having any effect on the in game rows and columns. I tried deleting the folder and I have launched three times since then and the jei config folder has not regenerated.
